### DISCUSSION TITLE
OpenGrok 1.7.39 is failed to get file diff for git SCM

### DISCUSSION THREAD

**Discussion by kribakarans** [oracle/opengrok#4107] [Development]
Opengrok 1.7.39 is failed to get the file difference for Git SCM. I was running Tomcat 10.0.27 and JDK 11.
Indexer command:
```
opengrok-indexer     -J=-Djava.util.logging.config.file=/opengrok/etc/logging.properties     -a /opengrok/dist/lib/opengrok.jar --     -c /usr/bin/ctags     -s /opengrok/src -d /opengrok/data -H -P -S -G     -W /opengrok/etc/configuration.xml -U http://localhost:8080/source
```
Once I ran the indexer command, the source is added in "Project" and it is not diplaying the git SCM type and remote URL.

```
root@virtualbox:/opengrok/src/cgit# git remote -v
origin	https://github.com/zx2c4/cgit.git (fetch)
origin	https://github.com/zx2c4/cgit.git (push)
```
Kindly help to resolve the issue.

**Comment by vladak** [comment]
The indexer arguments look sane. Anything interesting in the indexer logs ? Particularly in the beginning when repositories are detected.
